procedures/accounts_setup.md

```diff
@@ -0,0 +1,250 @@
+* MFA - More information on Mutliple-Factor-Authentication can be found in https://login.mozilla.com/duo/ which is what we use for LDAP/auth0 based logins. Follow [these](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=145590267) instructions to download the Duo Mobile app and set it up.
+* Mercurial - Most development in releng (and at Mozilla writ-large) is stored in version control using [hg](http://mercurial.selenic.com/). There is an excellent step-by-step guide for setting up and using hg: [Mercurial for Mozillians](https://mozilla-version-control-tools.readthedocs.org/en/latest/hgmozilla/index.html). More information on this can be found in [here](https://firefox-source-docs.mozilla.org/contributing/vcs/mercurial.html) and [here](https://firefox-source-docs.mozilla.org/contributing/contribution_quickref.html).
+There are 3 levels of commit access:
+a) Level 1 access allows you to use the [Try Server](https://wiki.mozilla.org/ReleaseEngineering/TryServer) and setup user repos. As a new contributor, you should request this on day one.
+b) Level 2 access is required to land code in the build and project repos. Once you have a proven track record of successful patches, you can ask your manager/mentor to vouch for your Level 2 access. Your manager/mentor can also land patches for you until you receive Level 2 access
+c) Level 3 access is required to land code in [mozilla-central](https://hg.mozilla.org/mozilla-central) and its derived integration & release branches. At some point in your Mozilla contribution story, you may need Level 3 access but many contributors never do. Talk to your manager/mentor if you think you need this access. You should already have Level 2 access when you request Level 3
+* Phabricator - you'll be granted access to this via SSO
+The root webview of the Mozilla hg repositories is here: https://hg.mozilla.org/ while most releng code lives in repos under https://hg.mozilla.org/build.
+Please fill out the [Commit Access form](https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/) and submit it via a bug on file such as [this](TODO help from Heitor here).
+
+## Git & Github
+Please file a bug similar to [bug 1726348](https://bugzilla.mozilla.org/show_bug.cgi?id=1726348) and include the following teams to be added. Ask for your manager to vouch in the bug for these changes. The teams you need to be added are:
+1. https://github.com/orgs/mozilla/teams/releng/members
+2. https://github.com/orgs/mozilla-releng/people
+3. https://github.com/orgs/mozilla-releng/teams/releng/members
+4. https://github.com/orgs/mozilla-mobile/teams/releng/members
+5. https://github.com/mozilla-services/cloudops-infra/ via filing a ticket [here](https://github.com/mozilla-services/github-management/issues/new?assignees=&labels=&template=NewOrgMember.md&title=) similar to [this](https://github.com/mozilla-services/github-management/issues/129).
```

Comment:
Should we mention [mozilla-services](https://github.com/orgs/mozilla-services/teams/releng/members) and [mozilla-mobile](https://github.com/orgs/mozilla-mobile/teams/releng/members)?

Reply:
https://github.com/orgs/mozilla-mobile/teams/releng/members is already mentioned at 4) whereas https://github.com/orgs/mozilla-services/teams/releng/members is theoretically granted via 5 but I think the latter can be stressed out as a separate team. Good point, adjusted.